.tekton/notifications-frontend-pull-request.yaml

```diff
@@ -219,7 +219,7 @@ spec:
         params:
           - name: url
             value: https://github.com/RedHatInsights/konflux-consoledot-frontend-build
-          - name: revision
+          - name: a6838b71b88dc1e84f11764c9734e4880096585d
```

Comment:
`name` should be `revision` and the `value` should be `a6838b71b88dc1e84f11764c9734e4880096585d`

Reply:
oop! good catch, thanks. updated!